### PR TITLE
[16.0][MIG] printer_zpl2: Migration to 16.0

### PR DESCRIPTION
**Pull request by QuocDuong1306**

### CODE REVIEW

**Review by micheledic — APPROVED** [review]

**Comment by luke-stdev001** [comment]
works as expected, anything blocking this from being merged?

File: printer_zpl2/models/printing_label_zpl2.py [hunk 1]
```diff
@@ -504,7 +504,9 @@ def _generate_labelary_image(self):
                 zpl_file = self._generate_zpl2_data(record, labelary_emul=True, **extra)
                 files = {"file": zpl_file}
                 headers = {"Accept": "image/png"}
-                response = requests.post(url, headers=headers, files=files, stream=True)
+                response = requests.post(
+                    url, headers=headers, files=files, stream=True, timeout=30
```
Comment (sebalix) [inline printer_zpl2/models/printing_label_zpl2.py:508]: This change should normally be outside of the migration commit as it's not tied to the Odoo migration at all, and could benefit to other Odoo versions.

Not blocking.

**Review by sebalix — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Let's move it anyway:

/ocabot migration printer_zpl2
/ocabot merge nobump

**Pull request merged**